examples/vitals/tesla.proto

```diff
@@ -43,17 +324,19 @@ message TeslaHardwareId {
 }
 
 message TeslaEnergyEcuAttributes {
-    optional int32 ecuType = 1;
+    int32 ecuType = 1;
     optional TeslaHardwareId hardwareId = 2;
+    optional PVInverterAttributes pvInverterAttributes = 3;
```

Comment:
I only ever see `ecuType` populated in my install, but the spec had these also in case others do return them.